release-notes/swan-lake-alpha4-release-note.md

```diff
@@ -56,8 +56,65 @@ If you have not installed Ballerina, then download the [installers](/downloads/#
 
 #### New Features
 
+##### Support for Relational Expressions With All Ordered Types
+
+Relational expressions (`<`, `>`, `<=`, and `>=`) are supported with all [ordered types](https://ballerina.io/spec/lang/draft/v2020-12-17/#ordering). The static type of both operands must belong to the same ordered type.
+
+##### Support for inferring the `typedesc` argument of a dependently-typed function from the contextually expected type
```

Comment:
```suggestion
##### Support for Inferring the `typedesc` Argument of a Dependently-Typed Function from the Contextually-Expected Type
```
This is too long for a heading. Either we can rephrase it or make it a bullet point.

Reply:
Will try and rephrase it. But I don't think we can make any arbitrary heading a bullet point. Don't we use bullet points only if we don't have to describe the change in detail + all the content under that particular area is specified as bullet points?

Reply:
Changed the capitalization, but as discussed offline, kept most of the heading as is since it's quite difficult to describe the feature by removing any of the other words.